linux/x86-64/python3/acrcloud/async_recognizer.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python
+#-*- coding:utf-8 -*-
+
+'''
+    @author qinxue.pan E-mail: xue@acrcloud.com
+    @version 1.0.0
+    @create 2015.10.01
+'''
+
+import os
+import sys
+import hmac
+import time
+import json
+import base64
+import hashlib
+import aiohttp
+import async_timeout
+import datetime
+
+import acrcloud_extr_tool
+
+'''
+Copyright 2015 ACRCloud Recognizer v1.0.0
```

Comment:
This is a new file, so either the name should be dropped from the copyright line or it should be updated.